### ISSUE TITLE
Add support for location in audit

### ISSUE THREAD

**Issue by yanokwa** [XLSForm/pyxform#253]
Building on the issue at https://github.com/opendatakit/xforms-spec/issues/219 and PR at https://github.com/opendatakit/xforms-spec/pull/221/files, we should add support for locations in the audit log.

My rough proposal is
* Add support for three parameters for audit questions (`location-priority`, `location-min-interval`, `location-max-age`) (note, we don't include the odk:)
* If one of the parameters are entered, then all three are required
* `priority` has four potential values: `no-power`, `low-power`, `balanced`, or `high-accuracy`
* `max-age` and `min-interval` are positive integers. `max-age` >= `min-interval`

Sample output looks like:
```xml
<bind nodeset="/orx:meta/orx:audit" type="binary" 
           odk:location-priority="balanced" odk:location-min-interval="10" odk:location-max-age="60" />
```

**Comment by yanokwa** [comment]
@KeynesYouDigIt This something you'd be willing to take on?

**Comment by ukanga** [comment]
This is closed by #253

**Comment by KeynesYouDigIt** [comment]
@yanokwa Sorry about that! missed your mention somehow.
